# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/linux/96-usb-hid-apple-kbd-mapping.patch
# Copyright (C) 2004 - 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

The Apple USB keyboards carry symbols differing from the on-wire values sent.
So we swap the events here simillar as in the 95-adb-mapping patch.
Likewise, default to the normal F* keys for usability on the Linux desktop.

  - Rene Rebe <rene@exactcode.de>

--- linux-2.6.30/drivers/hid/hid-apple.c.orig	2009-09-19 22:34:35.000000000 +0200
+++ linux-2.6.30/drivers/hid/hid-apple.c	2009-09-19 22:34:44.000000000 +0200
@@ -45,7 +45,7 @@
 MODULE_PARM_DESC(iso_layout, "Swap the backtick/tilde and greater-than/less-than keys. "
 		"([-1] = auto, 0 = disabled, 1 = enabled)");
 
-static unsigned int swap_opt_cmd;
+static unsigned int swap_opt_cmd = 1;
 module_param(swap_opt_cmd, uint, 0644);
 MODULE_PARM_DESC(swap_opt_cmd, "Swap the Option (\"Alt\") and Command (\"Flag\") keys. "
 		"(For people who want to keep Windows PC keyboard muscle memory. "
